.gitignore

```diff
@@ -1,12 +1,8 @@
 build
 phpunit.xml
 Resources/doc/_build/*
-nbproject
-.idea
-coverage
 composer.lock
-/vendor
-composer.phar
+vendor
```

Comment:
Doesn't this mean that `SonataAdminBundle/Resources/public/vendor/` is now ignored ?